gateway/modules/crud/bolt/create.go

```diff
@@ -0,0 +1,100 @@
+package bolt
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/boltdb/bolt"
+	"github.com/sirupsen/logrus"
+
+	"github.com/spaceuptech/space-cloud/gateway/model"
+	"github.com/spaceuptech/space-cloud/gateway/utils"
+)
+
+// Create inserts a document (or multiple when op is "all") into the database
+func (b *Bolt) Create(ctx context.Context, project, col string, req *model.CreateRequest) (int64, error) {
+	// Create a collection object
+	// todo check if exists
+	switch req.Operation {
+	case utils.One:
+		// Insert single document
+		if err := b.client.Update(func(tx *bolt.Tx) error {
+			count, _, err := b.Read(ctx, project, col, &model.ReadRequest{
+				Find:      req.Document.(map[string]interface{}),
```

Comment:
Just check if the `_id` exists. Don't check for the entire document, that is incorrect. What if the user insert two documents with the same `_id` but different `name` fields